tests/Support/Traits/AssertCss.php

```diff
@@ -88,12 +88,12 @@ private static function assertNotContainsCss(string $needle, string $haystack)
      * @param string $needle
      * @param string $haystack
      */
-    private static function assertContainsCssCount(
+    private function assertContainsCssCount(
         int $expectedCount,
         string $needle,
         string $haystack
     ) {
-        self::assertSame(
+        $this->assertSame(
```

Comment:
`assertSame` is a `static` method and should be called staticly.  Is there an issue with Psalm that we need a workaround for here?

Reply:
Looks like an issue with Psalm then.  `assertSame` is definitely a `static` method in PHPUnit.

I've added #792 to capture this.  For now, can we revert changes relating to calls via `self::` vs `$this` (and also any changes making methods `static` vs non-`static`), so that we can get this PR merged, and then move on to address the remaining issues separately?